app/models/king.rb

```diff
@@ -1,2 +1,19 @@
 class King < ChessPiece
+  X_MOVE_FACTOR = 1
+  Y_MOVE_FACTOR = 1
+
+  def valid_move?(destination_x, destination_y)
+    fail ArgumentError, 'Invalid Move' if obstructed?(destination_x, destination_y)
+    exceeds_x_movement?(destination_x) || exceeds_y_movement?(destination_y) ? false : true
```

Comment:
:+1: That is too funny...it is absolutely very readable (and, in ways, more readable), but it took my brain a second to see it. Do you mind me asking what your major was in college, because this looks super-engineering-y, whereas mine is straight from a literature major...I love that Ruby is flexible enough to allow developers to have their own style... :thought_balloon: